src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.exception.InValidInputException;
+import racingcar.model.RacingCar;
+import racingcar.model.RacingCars;
+import racingcar.view.RacingCarView;
+
+public class RacingCarController {
+
+    RacingCars racingCars;
+    RacingCarView racingCarView;
+
+    public RacingCarController(RacingCarView racingCarView) {
+        this.racingCarView = racingCarView;
+        this.racingCars = new RacingCars(requestCarNames());
+    }
+
+    public void playGame() {
+        int trial = racingCarView.requestTrial();
+        List<RacingCar> participants = racingCars.getRacingCars();
+
+        racingCarView.startGameRound();
+
+        for (int i = 0; i < trial; i++) {
+            playGameRound(participants);
+        }
```

Comment:
`racingCars` 객체는 이미 상태로 **participants**를 가지고있는데, 아래와 같이 racingCars 객체에게 메시지를 보내보면 어떨까요? 
```suggestion
        racingCarView.startGameRound();

        for (int i = 0; i < trial; i++) {
            racingCars.playGameRound();
        }
```

Reply:
view에 participants를 전달하기 위해 getter를 사용하고 있습니다.